packages/website/ts/components/inputs/allowance_toggle.tsx

```diff
@@ -74,9 +75,21 @@ export class AllowanceToggle extends React.Component<AllowanceToggleProps, Allow
             newAllowanceAmountInBaseUnits = DEFAULT_ALLOWANCE_AMOUNT_IN_BASE_UNITS;
         }
         try {
-            await this.props.blockchain.setProxyAllowanceAsync(this.props.token, newAllowanceAmountInBaseUnits);
+            await this.props.blockchain.setProxyAllowanceAsync(this.props.token, newAllowanceAmountInBaseUnits)
```

Comment:
Add missing semi-colon. Do you have a prettier extension installed in your text editor?

Reply:
Oops. I do not -- will fix and install